### PR TITLE
Updated to SDK 1.0.0-preview2-003131

### PR DESCRIPTION
**Pull request by thecodejunkie**

### CODE REVIEW

**Comment by jchannon** [comment]
Rebase req'd 😄

File: build.sh [hunk 1]
```diff
@@ -28,22 +28,12 @@ for i in "$@"; do
...
-  # set to 0 if not found
-  type $1 >/dev/null 2>&1 || { local return_=0; }
-  # return value
-  echo "$return_"
+  echo "Installing dotnet"
+  wget https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0-preview2/scripts/obtain/dotnet-install.sh
+  sudo bash dotnet-install.sh --version 1.0.0-preview2-003131 --install-dir .dotnet
+  export PATH=.dotnet:$PATH
+  export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
+  dotnet --info
```
Comment (jchannon) [inline build.sh]: We should add this to the build.ps1 too

Comment (thecodejunkie) [inline build.sh]: sure.. send a round of PRs and I'll review them :) But we should talk to @devlead first about creating a shim build.sh/ps1 for all the additional repos to cut down on the amount of repetitive work

Comment (thecodejunkie) [inline build.sh]: i.e either way I'm not going to include that change in this PR

**Comment by jchannon** [comment]
OK lets just wait for @devlead <https://github.com/devlead> to do his
stuff, lots of it.....

**Pull request merged**